src/VisualStudio/Tools/Test/DomainServiceWizard/BusinessLogicClassContextTests.cs

```diff
@@ -116,6 +117,18 @@ public void BusinessLogicClass_Context_EFDbContextTest()
             Assert.IsTrue(dbContext.NeedToGenerateMetadataClasses);
         }
 
+        //[TestMethod]
+        //[Description("Tests if LinqToEntitiesDbContext works properly for EFCore entities")]
+        //public void BusinessLogicClass_Context_EFCoreDbContextTest()
+        //{
+        //    // TODO: This won't work since LinqToEntitiesDbContext uses EF6 metadata model
+        //    // Something similar to LinqToEntitiesDbContext but using EF Core's IModel must be built
+        //    //LinqToEntitiesDbContext dbContext = new LinqToEntitiesDbContext(typeof(EFCoreModels.Northwind.EFCoreDbCtxNorthwindEntities));
+        //    //Assert.AreEqual(11, dbContext.Entities.Count());
+        //    //Assert.IsTrue(dbContext.NeedToGenerateMetadataClasses);
+        //    throw new NotImplementedException();
+        //}
+
```

Comment:
does it make sense to keep the code ? or is it better to just remove the modification?

Reply:
I think it makes sense since the TODO explains was must be done.
So I will keep it.